src/diffusers/pipelines/alt_diffusion/pipeline_alt_diffusion.py

```diff
@@ -106,10 +106,10 @@ class AltDiffusionPipeline(DiffusionPipeline, TextualInversionLoaderMixin, LoraL
         feature_extractor ([`~transformers.CLIPImageProcessor`]):
             A `CLIPImageProcessor` to extract features from generated images; used as inputs to the `safety_checker`.
     """
-
```

Comment:
the code quality test faill and I have run `make fix-copies` already

probably due to removing this line here, it is correct to remove I think since source pipeline did not have this line here. However this is not something I changed in this PR so wonder why the "fix-copies" fix this now? 

cc @DN6

Reply:
Facing this too but to have my sanity, I just manually corrected it.